docs/ref/models/querysets.txt

```diff
@@ -268,6 +268,51 @@ control the name of the annotation::
 For an in-depth discussion of aggregation, see :doc:`the topic guide on
 Aggregation </topics/db/aggregation>`.
 
+``alias()``
+~~~~~~~~~~~~~~
+
+.. method:: alias(*args, **kwargs)
```

Comment:
You we really allow `*args` to be passed here? It doesn't look like its tested below and it seems like using `alias` with the few number of expressions that implement default naming is unlikely.